components/cv_form/projects.tsx

```diff
@@ -186,7 +186,6 @@ export default function Projects({ fProps, setProjectsToRemove }: Props) {
                       className="flex text-indigo-500"
                       name={`projects[${projectsIndex}].ongoing`}
                       type="checkbox"
-                      checked={project.ongoing}
```

Comment:
We don't need checked prop since the formik controls the value with the `name` prop.